### PR TITLE
Add Ubuntu environment setup instructions to docs/getting-set-up

### PR DESCRIPTION
**Pull request by Sphar-io**
Added a set of environment setup instructions for Ubuntu to the getting-set-up docs page, which now provides instructions for Windows, Mac, and Ubuntu

# PR Checklist:

- [x] Have you run the unit, integration and smoke tests as described here? https://docs.corda.net/head/testing.html
- [x] If you added/changed public APIs, did you write/update the JavaDocs?
- [x] If the changes are of interest to application developers, have you added them to the changelog, and potentially release notes?
- [x] If you are contributing for the first time, please read the agreement in CONTRIBUTING.md now and add to this Pull Request that you agree to it.

Thanks for your code, it's appreciated! :)

### CODE REVIEW

**Comment by tlil** [comment]
BTW, there's also this open PR by @anthonyk https://github.com/corda/corda/pull/3578

File: docs/source/getting-set-up.rst [hunk 1]
```diff
@@ -95,6 +97,32 @@ IntelliJ
 2. Download and run the executable to install IntelliJ Community Edition (use the default settings)
 3. Ensure the Kotlin plugin in Intellij is updated to version |kotlin_version|
 
+.. _ubuntu-label:
+
+Ubuntu
+------
+
```
Comment (anthonykeenan) [inline docs/source/getting-set-up.rst]: Would you be able to add a note like there is for the Windows & Mac instructions here, and update the other notes to be include the Linux instructions.

**Comment by anthonykeenan** [comment]
Also feel free to add your name to https://github.com/corda/corda/blob/master/CONTRIBUTORS.md if you haven't already.

**Comment by anthonykeenan** [comment]
LGTM

**Pull request merged**